contracts/interfaces/IRelayHub.sol

```diff
@@ -107,14 +107,13 @@ interface IRelayHub {
 
     function isRelayManagerStaked(address relayManager) external view returns(bool);
 
-    // get the relay data from the manager address
-    function getRelayData(address relayManager) external view returns(RelayData memory relayInfo);
+    // get the relay info from the manager address
+    function getRelayInfo(address relayManager) external view returns(RelayManagerData memory relayManagerData);
 
-    // Represents the relay data for the relay
-    struct RelayData {
+    // Represents the relay data for a particular relay manager
+    struct RelayManagerData {
         address manager;
```

Comment:
i would have liked to remove this field, but it's useful on the javascript side.

the reason why i'd like not to have it is that it's redundant: the key to access this map will have the same value as this field. 

however, once an entry is received from the javascript side we no longer have the mapping around, so it's good to keep this value at hand.

we'd also need a way to signal whether a specific key has a value in the solidity mapping, and an `exists` bool field isn't ideal